packages/components/docs/localDev.md

```diff
@@ -172,14 +175,20 @@ merge back to develop.
 Steps for package version numbering during feature branch development:
 1. Create your feature branch off of develop, i.e. fb_feature_1, and add your changes.
 1. When you are ready to push an alpha version up to Artifactory so you can test it in your application and
-on TeamCity, run the `yarn publish` command from the @labkey/components package root directory. This command
-will then prompt you for a new alpha package version. Ex. if adding a new feature and the current version is `0.1.0`,
-you would use `0.2.0-fb-feature-1.1`.
+on TeamCity, edit the version of the @labkey/components package in `package.json`.
```

Comment:
Instead of editing the `package.json` file directly, I find it very useful to use `npm version prerelease --preid=X`, which does the editing for you. Do we want to recommend that here?

Reply:
Updated. See 06f52f3.